src/olympia/abuse/models.py

```diff
@@ -53,6 +54,7 @@ class CinderJob(ModelBase):
             'AMO_DISABLE_ADDON',
             'AMO_DELETE_RATING',
             'AMO_DELETE_COLLECTION',
+            'AMO_REJECT_VERSION_ADDON',
```

Comment:
Add a comment explaining that this action (and its appeal) is stricly handled by reviewers on AMO and not by moderators on Cinder, so it should never show up in the webhook payload